### DISCUSSION TITLE
Any recommendations for a Shelly i3-like device?

### DISCUSSION THREAD

**Discussion by afoeder** [Koenkk/zigbee2mqtt#7235] [Devices]
// edit: I've just realized that the battery-driven Hue Wall Switch is capable of this, so I'll go with this one.

Hey zigbees, do you know any devices that I can wire to actual light switches and that only communicate the switch status? Like exactly what [Shelly's i3](https://shelly.cloud/products/shelly-i3-smart-home-automation-device/) is doing, but via zigbee.
Of course it can't hurt if it would also have a relay built in but that's particularly not necessary.

Background: I want to replace switches with Busch-Jaeger ones but I want to go easy on the [LightLink](https://www.zigbee2mqtt.io/devices/6735_6736_6737.html) devices for pricing reasons. But I want the series because of aesthetics (`dynasty` for those who're curious). So I could start with classical switches made smart which would be in sum a bit cheapter.

More context: I already have four pairs of LightLink control panels with relays, mains and dim modules underneath and now it's the bedroom who's next, but since the entry and both bed sides have switches and mains outlets I need to cut on the budget a bit.
Thanks for any reco's!

**Comment by vdkeybus** [comment]
You can ask Shellies to announce and publish on the MQTT server, giving the same functionality.

I'm using the 1PM and 2PM (18 in total) since they have a relay that I normally keep on, but it can be pulsed e.g. to reset and re-pair Zigbee devices and avoid walking around. They are also available with UL rating. I run them in 'detached switch' state on MQTT, but if something goes wrong I can quickly put them in 'toggle switch' and operate my lights normally again.

The only caveat I have is that it needs a star-type network to a Wifi AP (so no meshing), and the units are sensitive to orientation in the wall box.

**Reply by afoeder** [reply]
I was looking particularly for Zigbees because of the "binding" feature. I had two situations already where my z2m on Home Assistant somehow got stuck and I want to avoid that basic infrastructure (simple ceiling lights) are affected by this; at least I want to minimize the impact.

**Reply by vdkeybus** [reply]
I see. With 'toggle switch' I basically return to turning on and off the supply of the light in that case. All lamps are designed to turn on when they receive power.

TBH I have more trouble with HA than with Z2M.

Didn't know about the Hue wall switch. _Quite_ expensive and tricky to replace the battery (whole switch has to be taken off).

There's also the Niko wall switch that has no batteries. it is powered by the piezo-electric switch itself (https://www.niko.eu/en/article/130-91004), but it is a Green device that isn't supported by z2m (yet). Works with Hue though, and comparable in price.

**Reply by afoeder** [reply]
yeah "unfortunately" I want to stick with the Busch-Jaeger devices in a two-way-connection/cross circuit, that's why I need a mix of the Busch-Jaeger Lightlink and something I can use at the point of the other switches.

_ah I have it open already_

![image](https://user-images.githubusercontent.com/1125168/116891056-91d41300-ac2e-11eb-80c3-7e653e254114.png)

The left switch will be substituted with a LightLink on a 6710 mains unit, the other switches are short circuit'ed in the wall; but I want to maintain their presence and functionality.

**Reply by poseva** [reply]
> I see. With 'toggle switch' I basically return to turning on and off the supply of the light in that case. All lamps are designed to turn on when they receive power.
> 
> TBH I have more trouble with HA than with Z2M.
> 
> Didn't know about the Hue wall switch. _Quite_ expensive and tricky to replace the battery (whole switch has to be taken off).
> 
> There's also the Niko wall switch that has no batteries. it is powered by the piezo-electric switch itself (https://www.niko.eu/en/article/130-91004), but it is a Green device that isn't supported by z2m (yet). Works with Hue though, and comparable in price.

How do you 'toggle switch' ? do you press on a button twice and then the Shelly will return to also turn on/off the relay than just communicate the state ? or ?

Thank you in advance!

**Comment by froogl** [comment]
I think the Hue Wall Switch module cannot do direct bindings, you have to use rules. And I personally would get a powered in-wall module like the Sunricher module (SR-ZG2833PAC-FOH). The Sunricher module also cannot do direct bindings (because it's a GP device), but at least you won't have to change batteries in a few years.

In the end your Busch-Jager ZLL devices are probably still the best option. They look great, have great features (direct bindings supported) and there are mains-powered and battery-operated options. Yes, they are pricy, but thanks to direct bindings they are super reliable.

**Comment by vdkeybus** [comment]
You can select 'toggle switch' function in the Shelly's web interface.


Op vr 26 nov. 2021 om 07:56 schreef Sebastian ***@***.***>:
